fuel-vm/src/interpreter/executors/instruction.rs

```diff
@@ -759,8 +759,9 @@ where
 
             Instruction::LDC(ldc) => {
                 let (a, b, c) = ldc.unpack();
-                self.dependent_gas_charge(self.gas_costs().ldc, r!(c))?;
-                self.load_contract_code(r!(a), r!(b), r!(c))?;
+                let len = r!(c);
+                self.load_contract_code(r!(a), r!(b), len)?;
+                self.dependent_gas_charge(self.gas_costs().ldc, len)?;
```

Comment:
If this isn't a material change, I might suggest reverting it and keeping it as is.

Reply:
It's a pattern we use on other opcodes. I don't think it will make a difference in performance, but it's a little clearer IMO.